scripts/mac_refresh.py

```
@@ -0,0 +1,92 @@
+def refresh_mac_slack():
+    home = Path.home()
+    config_dir = home / ".config/slack"
+    config_dir.mkdir(parents=True, exist_ok=True)
+    
+    print("🔄 Dynamic Scan: Authenticating and extracting secure Mac Slack session...")
+    cookie_path = home / "Library/Application Support/Slack/Cookies"
+    key = get_mac_slack_key()
+    d_cookie = None
+    
+    if cookie_path.exists() and key:
+        temp_db = "/tmp/slack_cookies"
```

Ruff (0.15.15)

[error] Probable insecure usage of temporary file or directory: "/tmp/slack_cookies"

(S108)